AnnoyancesFilter/sections/popups.txt

```diff
@@ -119,6 +119,7 @@ timesofindia.indiatimes.com##iframe[src^="https://timesofindia.indiatimes.com/ad
 quirkbooks.com#$#.newsletterPrompt { display: none !important; }
 quirkbooks.com#$?#html { overflow: visible !important; touch-action: auto !important; }
 bing.com##.ev_talkbox_wrapper
+microsoftedgewelcome.microsoft.com###popup-1000
```

Comment:
I'm not sure about blocking this popup. It's a tip for Edge users.

Reply:
It is still a modal popup though.

Reply:
Ah, wait a second, it duplicates the content from the main page. Well, ok.